scripts/select-random-player/select_random_player.lua

```diff
@@ -0,0 +1,30 @@
+local BE_SPECIAL = "Traitor"
+
+function selectRandomPlayer(be_special)
```

Comment:
Does this need to be global? You should prefer `local function`

Reply:
Yeah, description of PR explains it

> The function is public because auxlua offers a functionality to call functions from the global table panel. This lets you call the function without needing to rerun the script.

Reply:
:+1:

Reply:
![image](https://github.com/tgstation/auxlua-cookbook/assets/37270891/807a8158-fda8-4e76-aa5e-d438423dbf36)

Reply:
In that case you might need a selene allow here. I think `-- selene: allow(unscoped_variables)` will work, again install the selene extension to check

Reply:
I do have the selene extension but I'm not getting any errors with the current code.
I know that selene is working because specific things I write in code will definitely be linted.
I'll add the selene allow anyways